master/getting-started/kubernetes/installation/hosted/index.md

```diff
@@ -87,6 +87,20 @@ To use these manifests with a TLS enabled etcd cluster you must do the following
   - `etcd_key: /calico-secrets/etcd-key`
   - `etcd_cert: /calico-secrets/etcd-cert`
 
+### Authorization Options
+
+All of Calico's manifests assign the Pods two serviceaccounts.
```

Comment:
I think this section should spell out a little more clearly when users will want to do this.

How does a user know if their cluster needs RBAC or not?